builtin/logical/pki/backend_test.go

```diff
@@ -114,7 +114,7 @@ func TestPKI_RequireCN(t *testing.T) {
 	b, s := CreateBackendWithStorage(t)
 
 	resp, err := CBWrite(b, s, "root/generate/internal", map[string]interface{}{
-		"common_name": "myvault.com",
+		"common_name": "mybao.com",
```

Comment:
Hmm, if we're going to go to a different domain, let's do it in a separate PR if that's alright.

I'd probably also use `example.com` as the RFC 2606 specifies :-)

Reply:
Alright, lem fix that and I'll open up a PR shortly! :)